examples/evm/call-contract-with-token/CallContractWithToken.sol

```diff
@@ -38,6 +60,15 @@ contract DistributionExecutable is AxelarExecutable {
         gateway.callContractWithToken(destinationChain, destinationAddress, payload, symbol, amount);
     }
 
+    /**
+     * @notice logic to be executed on dest chain
+     * @dev this is triggered automatically by relayer
```

Comment:
I don't know if this is helpful

Reply:
I think it's worth having the distinction between logic on src vs logic on dest chain. As for the triggered automatically by relayer line I was hoping this would answer the question people might have on when/how they trigger it (i.e. that they don't need to).